cmd/vulcan-retirejs/main.go

```diff
@@ -355,6 +396,16 @@ func resolveTarget(target string) (string, error) {
 		Transport: tr,
 		Timeout:   timeout,
 	}
+	if assetType == "WebAddress" {
+		resp, err := client.Get(target)
+		if err == nil {
+			t := resp.Request.URL.String()
+			if resp.Request.URL.Path == "" {
+				t = fmt.Sprintf("%s/", t)
+			}
+			return t, nil
+		}
+	}
```

Comment:
We need to handle differently both execution branches: or is `WebAddress` or is `Hostname`, exclusively. For handling I mean doing the appropriate checks and error handling.

Reply:
You are completely right. Fixed in [7b37b77](https://github.com/adevinta/vulcan-checks/pull/145/commits/7b37b77899901e8d92e794d5a96ebfaa8b0aaae8)